services/application/src/actions/user/send-login-link.js

```diff
@@ -56,24 +56,34 @@ module.exports = async ({
   const addressValues = addressFields.map(field => company[field]).filter(v => v).map(stripLines);
   const supportEmail = context.email || app.email || company.supportEmail;
   const language = context.language || app.language || 'en-us';
+
   if (supportEmail) addressValues.push(supportEmail);
 
   const { token } = await createLoginToken({ applicationId, email: user.email, data: { source } });
   let url = `${authUrl}?token=${token}`;
   if (redirectTo) url = `${url}&redirectTo=${encodeURIComponent(redirectTo)}`;
 
+  const loginLinkTemplate = {
+    ...app.loginLinkTemplate.toObject(),
+    ...context.loginLinkTemplate.toObject(),
```

Comment:
Based on L52 above, it's possible the the `context` object, in some cases, will be a POJO, not a Mongoose object. As such, the `toObject` call will throw an error... a somewhat "hacky fix" would be to add a `toObject` function to the empty fallback on L52 that returns an empty object.... `{ toObject: ()  => ({}) };` this is probably the quickest way to deal with it